### PR TITLE
Transparent mirror support (issues resolved)

### PR DESCRIPTION
**Pull request by globalcitizen**
Resolving  issues from pull request #58

### CODE REVIEW

**Comment by globalcitizen** [comment]
My interpretation is that the fail is a CI bug.

File: build.sh [hunk 1]
```diff
@@ -32,5 +32,16 @@ if [[ -n "${SUFFIX}" ]]; then
 	SUFFIX="-${SUFFIX}"
 fi
 
-docker build --build-arg ARCH="${ARCH}" --build-arg MICROARCH="${MICROARCH}" --build-arg BOOTSTRAP="${BOOTSTRAP}" --build-arg SUFFIX="${SUFFIX}"  -t "${ORG}/${TARGET}:${VERSION}" -f "${NAME}.Dockerfile" .
+# Load local mirror as DIST
+DISTBUILDARG=''
+if [[ -e /etc/portage/make.conf ]]; then
+	export DIST=`portageq envvar GENTOO_MIRRORS|cut -d '"' -f 2|cut -d ' ' -f1`
```
Comment (ultrabug) [inline build.sh]: sorry if I'm missing something obvious, but what are those grep here for?

File: build.sh [hunk 2]
```diff
@@ -32,5 +32,16 @@ if [[ -n "${SUFFIX}" ]]; then
...
+	if [ ${DIST} ]; then
+		export DIST="${DIST}/releases/${ARCH}/autobuilds"
+		echo "DIST: using locally preferred mirror: '${DIST}'."
+	fi
+	DISTBUILDARG="--build-arg DIST=${DIST}"
```
Comment (ultrabug) [inline build.sh]: why not always pass the `--build-arg` and get a default on `DIST` on line 36? we could get rid of `DISTBUILDARG`

**Comment by ultrabug** [comment]
Closing, feel free to reopen if you think it's needed mate

Thank you again for your contribution and help!

**Pull request closed without merging**